app/configureStore.js

```diff
@@ -2,23 +2,18 @@
  * Create the store with dynamic reducers
  */
 
-import { createStore, applyMiddleware, compose } from 'redux';
+import { configureStore, getDefaultMiddleware } from '@reduxjs/toolkit';
 import { routerMiddleware } from 'connected-react-router';
 import { createInjectorsEnhancer, forceReducerReload } from 'redux-injectors';
 import createSagaMiddleware from 'redux-saga';
 import createReducer from './reducers';
 
-export default function configureStore(initialState = {}, history) {
-  let composeEnhancers = compose;
+export default function configureAppStore(initialState = {}, history) {
   const reduxSagaMonitorOptions = {};
 
   // If Redux Dev Tools and Saga Dev Tools Extensions are installed, enable them
   /* istanbul ignore next */
   if (process.env.NODE_ENV !== 'production' && typeof window === 'object') {
-    /* eslint-disable no-underscore-dangle */
-    if (window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__)
-      composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__({});
```

Comment:
This is no longer necessary for the Redux dev tools to work? Or is it simply handled inside Redux Toolkit?

If it's the latter, the parent `if` statement is now empty. Since the saga dev tools don't seem to be maintained + they have limited usability, would you mind just deleting that whole `if`?

(Same in the internals version of this file.)

Reply:
RTK already handles this internally.  Our `configureStore()` uses the `redux-devtools-extension` package, which wraps the global window function lookup.